app/src/main/java/io/intrepid/contest/rest/ContestStatusResponse.java

```diff
@@ -1,6 +1,37 @@
 package io.intrepid.contest.rest;
 
 public class ContestStatusResponse {
-    public boolean waitingForSubmissions;
-    public int numSubmissionsMissing;
+    private boolean submissionsEnded;
+    private int entriesSubmitted;
+    private int totalEntries;
+
+    private boolean contestEnded;
+    private int judgesSubmitted;
+    private int totalJudges;
+
+    public ContestStatusResponse(boolean submissionsEnded, boolean contestEnded,
```

Comment:
Not necessarily a commentary right now but a couple of these parameters could probably be moved to setters. 6 arguments is a bit too much